.github/workflows/backport-pull-request.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: backport-pull-request
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    env:
+      backport-label: apply-
```

Comment:
I chose a name, which is in particular different from the key of line 31, in order to avoid any chances to get name conflicts on the one hand and to not raise the impression that this would be the only place of usage.